blueflood-elasticsearch/pom.xml

```diff
@@ -71,13 +65,6 @@
       <scope>test</scope>
     </dependency>
 
-    <dependency>
-      <groupId>junit</groupId>
-      <artifactId>junit</artifactId>
```

Comment:
JUnit dependency was removed because of the reason you mentioned. 

I don't have the prior coverage. But, right now code coverage on files that I touched are below:
- AbstractElasticIO               - 88% (Line Coverage)
- ElasticIO                             - 81% (Line Coverage)
- ElasticsearchRestHelper   - 92% (Line Coverage)
- EventElasticsearchIO .      - 80% (Line Coverage)